centralstore/smartwrapper_test.go

```diff
@@ -66,6 +66,8 @@ func getAndStartSmartWrapper(storetype string) (*SmartWrapper, func(), error) {
 			DroppedSize:       1000,
 			SizeCheckInterval: duration("1s"),
 		},
+		// by default redis only has 16 database index slots available
+		GetRedisDatabaseVal: rand.Intn(16),
```

Comment:
Will this fail if two tests randomly choose the same db val?

Reply:
Yes. In the longer term, maybe we need to have better namespace mechanism for our test data or we use miniredis. As if right now, this is the easiest fix I could come up with